vortexasdk/client.py

```diff
@@ -66,7 +67,13 @@ def _create_url(self, path: str) -> str:
         return f"{API_URL}{path}?apikey={self.api_key}"
 
 
-def _send_post_request_data(offset, url, payload, size):
+def _send_post_request_data(offset, url, payload, size, progress_bar: tqdm):
+    # noinspection PyBroadException
+    try:
+        progress_bar.update(size)
+    except Exception:
+        logger.warn("Could not update progress bar")
+
     return _send_post_request(url, payload, size, offset)["data"]
```

Comment:
I can see this isnt part of this PR, but I find this syntax a little unintuitive.  Might be cleaner to do:
```
response = _send_post_request(...)
return response["data"]
```

Reply:
Good shout - I'm planning on refactoring the client over the next few days, I think the whole thing is messy.